tests/python/contrib/test_tedd.py

```diff
@@ -14,9 +14,13 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from tvm import te
-import numpy as np
 import re
+
+import numpy as np
+
+import pytest
```

Comment:
Is this needed?

Reply:
Good point. Numpy was there before and didn't notice it wasn't used. I tried using pytest.importorskip, but it didn't do the trick. in the end, that is a leftover. I'll fix it.